packages/@react-spectrum/list/src/listview.css

```diff
@@ -59,6 +59,10 @@
           background-color: var(--spectrum-table-row-background-color-selected-hover);
         }
 
+        &:focus-ring {
+          background-color: var(--spectrum-table-row-background-color-selected-key-focus);
+        }
```

Comment:
note this is in the emphasized styles, will have to double check the non-emphasized styles when we actually support a non-emphasized ListView